app/frontend/src/cljs/teet/meeting/meeting_view.cljs

```diff
@@ -881,27 +952,60 @@
         [:a {:href (common-controller/query-url :meeting/download-pdf
                                                 {:db/id (:db/id meeting)})
              :target "_blank" } "Download"]]]]
+     [:div {:class (<class common-styles/heading-and-action-style) :style {:margin-bottom "1rem" :padding "1rem 0 1rem 0" :border-bottom (str "1px solid " theme-colors/gray-lighter)}}
+      [typography/Heading2 (tr [:fields :meeting/agenda])]
+       (when edit?
+         [form/form-modal-button {:form-component   [agenda-form e! meeting]
+                                  :id               "add-agenda"
+                                  :form-value       {:meeting.agenda/responsible (select-keys user [:db/id
+                                                                                                    :user/id
+                                                                                                    :user/given-name
+                                                                                                    :user/family-name
+                                                                                                    :user/email
+                                                                                                    :user/person-id])}
+                                  :modal-title      (tr [:meeting :new-agenda-modal-title])
+                                  :button-component [buttons/button-primary {:start-icon (r/as-element
+                                                                                           [icons/content-add])} (tr [:meeting :add-agenda-button])]}])]
      [:div {:style {:margin-bottom "1rem"}}
       (doall
        (for [{id :db/id
-              :meeting.agenda/keys [topic responsible body decisions] :as agenda-topic} agenda]
+              :meeting.agenda/keys [topic responsible body decisions file/_attached-to] :as agenda-topic} agenda]
```

Comment:
Cant use `file/_attached-to` in `:meet.agenda/keys`... that would make the key `:meeting.agenda/file/_attached-to` which is not correct.

This needs to be as separate destructuring in the parent map.